### DISCUSSION TITLE
Help my mom set up a github account under my name and email

### DISCUSSION THREAD

**Discussion by betty997** [community/community#57133] [New to GitHub] [answered]
I know nothing about coding/computer programming but I came across github i entered my email and said forgot password so it sent me an email to change it I logged in bc I never remembered setting up an account!  Interestingly enough it showed my account with repositories on it!! Idk what this means but I knew I didn’t write it !! So I deleted it!! Sudenley on my photos on my phone I’m seeing all my pictures I thought were lost forever. I was happy until the next day all my pictures were no longer labeled with peoples faces/names and many of my photos no longer had locations on them either some were missing or hidden under an image!!  I’m sure my mom hacked my phone and using GitHub for the wrong purpose and made it appear to be me using this GitHub service!!  Just out of curriosity I signed in just now and GitHub requested an additional code which I entered in then it said GitHub is requesting access to my account ?  I honestly don’t understand how to use github and would like to ask   For my account to be deleted from any actions made and then be permanently removed!  [redacted due to PII]

**Comment by Bibiwei-Pere** [comment, marked as answer]
Hi! @betty997 
Are you requesting for your GitHub account to be permanently deleted?

If Yes, head over to settings > account
Scroll down and you'll see the option to permanently deleted your GitHub account.
Thanks.

**Comment by ghost** [comment]
I'm having the same issue

**Comment by Bibiwei-Pere** [comment]
Hi @282uigig 

Your question as already been answered above.
Please follow the steps above to delete your account.

Thanks!

**Reply by ghost** [reply]
How was this account created?

**Comment by emenemi** [comment]
no
